nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +255,34 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param receivers The list of receivers.
+   * @param field The class field element.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAndBase(List<Element> receivers, Element field) {
+    List<AccessPathElement> elements = new ArrayList<>();
+    elements.add(new AccessPathElement(field));
+    Root root;
+    Element rootReceiver;
+    if (receivers == null || receivers.size() == 0) {
```

Comment:
Do we rely on this behavior of treating a null `receivers` list the same as if it were `[this]`?  Maybe we should just disallow a null `receivers` parameter?  In any case, we need to document the handling of the null or 0-length receivers list in the Javadoc